hermes.js

```diff
@@ -78,7 +79,7 @@ var hermesKeyList = (function () {
             continue;
         }
         res.push(k);
-        if (k === hermesKey) {
+        if (k === hermesKey) {
```

Comment:
i'm confused what the change is here

Reply:
Weirdly, it was an NBSP previously. Webstorm flagged it.